src/promptflow-tools/promptflow/tools/open_source_llm.py

```diff
@@ -821,9 +990,16 @@ def call(
         content_formatter = ContentFormatterFactory.get_content_formatter(
             model_family=self.model_family,
             api=api,
-            chat_history=prompt
+            chat_history=prompt,
+            endpoint_url=self.endpoint_uri
         )
 
+        if isinstance(content_formatter, ServerlessLlamaContentFormatter):
+            if api == API.CHAT:
```

Comment:
people using custom connection may already provide these suffixes. Can you check if the url already does not contains them before adding them here.